enterprise/server/webhooks/github/github.go

```diff
@@ -200,13 +201,14 @@ func parsePullRequestOrReview(event interface{}) (*interfaces.WebhookData, error
 	}
 	isFork := v["PullRequest.Base.Repo.CloneURL"] != v["PullRequest.Head.Repo.CloneURL"]
 	return &interfaces.WebhookData{
-		EventName:     webhook_data.EventName.PullRequest,
-		PushedRepoURL: v["PullRequest.Head.Repo.CloneURL"],
-		PushedBranch:  v["PullRequest.Head.Ref"],
-		SHA:           v["PullRequest.Head.SHA"],
-		TargetRepoURL: v["PullRequest.Base.Repo.CloneURL"],
-		TargetBranch:  v["PullRequest.Base.Ref"],
-		IsTrusted:     !isFork,
+		EventName:          webhook_data.EventName.PullRequest,
+		PushedRepoURL:      v["PullRequest.Head.Repo.CloneURL"],
+		PushedBranch:       v["PullRequest.Head.Ref"],
+		SHA:                v["PullRequest.Head.SHA"],
+		TargetRepoURL:      v["PullRequest.Base.Repo.CloneURL"],
+		IsTargetRepoPublic: v["PullRequest.Base.Repo.Private"] == "false",
```

Comment:
can you define a bool for this above (just for clarity)?

Reply:
Done